rebar.config

```diff
@@ -136,7 +136,8 @@
                     bbmustache,
                     {hackney, "1.12.1"},
                     {websocket_client, ".*", {git, "git://github.com/aeternity/websocket_client", {ref, "a4fb3db"}}}
-                ]}
+                ]},
+                {ct_opts, [{create_priv_dir, auto_per_tc}]}
```

Comment:
I see http://www.rebar3.org/docs/configuration#section-common-test and http://erlang.org/doc/man/ct.html#run_test-1

Reply:
What do you mean?

Reply:
I mean that I saw that the change makes sense.

Actually a better reference is http://erlang.org/doc/apps/common_test/write_test_chapter.html#id78698